### PR TITLE
add /join and /discord as redirect to discord

### PR DESCRIPTION
**Pull request by jierlich**
Add redirect links to discord for ease of access.

As a heads up, I can't run locally without updating `caniuse-lite`. Should I include the update for this here as well?

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -50,6 +50,8 @@ import createTheme from 'utils/createCustomTheme'
 import graphqlClient from 'utils/graphql'
 import { RewardsProvider } from './contexts/Rewards'
 
+const DISCORD_LINK = 'https://discord.com/invite/74xXr57XHT'
```
Comment (dalberto) [inline src/App.tsx:53]: @controtie We should consider where we want to put constants like this. `App.tsx` seems like a suboptimal place.

Comment (jierlich) [reply src/App.tsx:53]: Good point. It looks like `src/constants` already exists. What are your thoughts on a `src/constants/externalLinks.ts` file?

Comment (dalberto) [reply src/App.tsx:53]: That seems like a good place, but I defer to @controtie.

Comment (controtie) [reply src/App.tsx:53]: src/constants/externalLinks sounds great!

Comment (jierlich) [reply src/App.tsx]: Cool! Just pushed a change adding that file + a small stylistic cleanup

**Review by 0xModene — APPROVED** [review]
LGTM

I went ahead and reviewed this as this was mentioned on the DeFridays call and wanted to make sure it was out there

**Pull request merged**